Model/Cron.php

```diff
@@ -2243,4 +2243,15 @@ private function addNotificationErrorComment($errorMessage)
             $this->_order->save();
         }
     }
+
+    /**
+     * Checks if a payment is wallet payment method
+     * @param $notificationPaymentMethod
+     * @return bool
+     */
+    private function isWalletPaymentMethod($notificationPaymentMethod): bool
+    {
+        $walletPaymentMethods = array('googlepay', 'paywithgoogle', 'wechatpayWeb', 'amazonpay', 'applepay');
```

Comment:
Could this array be moved to a constant in the `Helper/PaymentMethods.php` class?